packages/miew/.stylelintrc.js

```diff
@@ -7,6 +7,6 @@ module.exports = {
     'alpha-value-notation': 'number',
     'property-no-vendor-prefix': null,
     'declaration-block-no-redundant-longhand-properties': null,
-    'scss/at-import-partial-extension': 'always',
```

Comment:
Thank you for your interest in improving Miew!

Unfortunately, it appears that the rule you've suggested is no longer supported in stylelint-config-recommended-scss v14.1.0. We currently use stylelint-config-standard-scss v^13.1.0, which relies on an earlier version of the recommended rules. As a result, we're unable to incorporate this specific rule change at this time.